### DISCUSSION TITLE
How use license-report in a lerna monorepo project

### DISCUSSION THREAD

**Discussion by geetee24** [ironSource/license-report#154] [Q&A]
Need license-report  to traverse each repo under the projects folder and generate a single output of unique licenses.  Lerna in use.

How to?

**Comment by BePo65** [comment]
>  a single output of unique licenses
1. does this mean that you want 1 report for the complete monorepo without duplicate entries?

2. as I don*t have a monorepo at hand (and do not use Lerna by now): can you create a sample monorepo (best on github) that I can use to try to find a solution?

**Comment by geetee24** [comment]
for now I am calling it in a loop as I enum it through the packages
folder.  Is weak but works.

**Comment by BePo65** [comment]
my idea was to look, if I could use / modify [license-report-recursive](https://github.com/bepo65/license-report-recursive) for that purpose (it is a wrapper around license-report to iterate through the complete dependency tree).

But I need a sample to try it out - for that I would appreciate it if you could prepare a monorepo with dummy dependencies.
